docs/IntegrationWithExistingApps.md

```diff
@@ -782,3 +803,13 @@ if (!foundHash) {
   display('platform', isMac ? 'objc' : 'android');
 }
 </script>
+
+## Creating a release build in Android Studio
+
+You can use Android Studio to create your release builds too! It’s as easy as creating release builds of your previously-existing native Android app. There’s just one additional step, which you’ll have to do before every release build. You need to execute the following to create a React Native bundle, which’ll be included with your native Android app:
+
+    $ react-native bundle --platform android --dev false --entry-file index.android.js --bundle-output android/com/your-company-name/app-package-name/src/main/assets/index.android.bundle --assets-dest android/com/your-company-name/app-package-name/src/main/res/
```

Comment:
Applying the gradle script will work - `apply from: "../../node_modules/react-native/react.gradle"`

You can create a release build via the command line: `./gradlew assembleRelease`

or if you want to use Android studio you need to configure a few settings:
- Preferences -> "Build, Execution, Deployment" -> Compiler -> "Configure on demand" = Uncheck
- Mac only - Change path variable for GUI apps - `sudo launchctl config user path $PATH`

The configure on demand setting in Android studio causes the bundle task to skip. I'll read up a bit more on how to make it work with this setting.